lib/taskana-core/src/test/java/acceptance/classification/CreateClassificationAccTest.java

```diff
@@ -1,297 +1,374 @@
+    assertThatCode(() -> classificationService.createClassification(classificationCopy))
+        .doesNotThrowAnyException();
   }
 
-  @WithAccessId(user = "taskadmin")
-  @WithAccessId(user = "user-1-1")
-  @TestTemplate
-  void should_ThrowException_When_UserRoleIsNotAdminOrBusinessAdmin() {
-    ClassificationImpl classification =
-        (ClassificationImpl) CLASSIFICATION_SERVICE.newClassification("newKey718", "", "TASK");
+  @WithAccessId(user = "businessadmin")
+  @TestFactory
+  Stream<DynamicTest>
+      should_SetDefaultServiceLevel_When_TryingToCreateClassificationWithMissingServiceLevel() {
+    Iterator<String> iterator = Arrays.asList("", null).iterator();
+    AtomicInteger i = new AtomicInteger();
+
+    ThrowingConsumer<String> test =
+        serviceLevel -> {
+          Classification classification =
+              classificationService.newClassification("Key6_" + i.getAndIncrement(), "", "TASK");
+          classification.setServiceLevel(serviceLevel);
 
-    ThrowingCallable createClassificationCall =
-        () -> {
-          CLASSIFICATION_SERVICE.createClassification(classification);
+          classification = classificationService.createClassification(classification);
+
+          assertThat(classification.getServiceLevel()).isEqualTo("P0D");
         };
 
-    assertThatThrownBy(createClassificationCall).isInstanceOf(NotAuthorizedException.class);
+    return DynamicTest.stream(iterator, c -> String.format("for '%s'", c), test);
   }
 
-  @WithAccessId(user = "admin")
+  @WithAccessId(user = "businessadmin")
   @Test
-  void should_SetDefaultServiceLevel_When_TryingToCreateClassificationWithMissingServiceLevel()
-      throws Exception {
-    Classification classification =
-        CLASSIFICATION_SERVICE.newClassification("newKey718", "", "TASK");
-    classification = CLASSIFICATION_SERVICE.createClassification(classification);
-    assertThat(classification.getServiceLevel()).isEqualTo("P0D");
+  void should_SetDefaultValues_When_CreatingClassificationWithoutSpecificValues() throws Exception {
+    Classification classification = classificationService.newClassification("Key7", "", "TASK");
+    classification = classificationService.createClassification(classification);
 
-    classification = CLASSIFICATION_SERVICE.newClassification("newKey71", "", "TASK");
-    classification.setServiceLevel("");
-    classification = CLASSIFICATION_SERVICE.createClassification(classification);
     assertThat(classification.getServiceLevel()).isEqualTo("P0D");
+    assertThat(classification.getId()).isNotEqualTo(null);
+    assertThat(classification.getId()).isNotEmpty();
+    assertThat(classification.getCreated()).isNotNull();
+    assertThat(classification.getModified()).isNotNull();
+    assertThat(classification.getParentId()).isEqualTo("");
+    assertThat(classification.getParentKey()).isEqualTo("");
+    assertThat(classification.getIsValidInDomain()).isEqualTo(false);
   }
```

Comment:
Is this a dupliacate test szenario? What do you think?

Reply:
I think i just feared to delete a test scenario from the old version (...WithMissingServiceLevel). But I see that it's not necessary. We can delete it. Do you agree?